pages/Announcements/Overview.md

```diff
@@ -30,18 +30,36 @@ Most serializations use outside standards, but some require additional clarifica
 
 ### hexadecimal
 
+Used to represent bytes.
+
 - MUST use 0-9,a-f representation
 - MUST be lowercase
 - MUST be prefixed with a `0x`
-- MUST NOT have spaces
-- MUST NOT have any zero padding
+- MUST NOT have spaces or separators
+- MUST have two characters per byte
```

Comment:
I wonder if we should say "MUST have two characters per byte in addition to the 0x characters"

Reply:
No harm in being completely pedantic in one's language in a spec